terraform/modules/lev_api/outputs.tf

```diff
@@ -1,3 +1,19 @@
 output "service_url" {
   value = aws_apprunner_service.lev_api.service_url
 }
+
+output "lev_rds_db_username" {
+  value = aws_rds_cluster.lev_rds_postgres_cluster.master_username
+}
+
+output "lev_rds_db_password" {
+  value = aws_rds_cluster.lev_rds_postgres_cluster.master_password
+}
+
+output "lev_rds_db_name" {
+  value = aws_rds_cluster.lev_rds_postgres_cluster.database_name
+}
+
+output "lev_rds_db_host" {
+  value = aws_rds_cluster.lev_rds_postgres_cluster.endpoint
+}
```

Comment:
I'm still quite new to terraform, so this might be a very bad way to share credentials - very happy for input here 😅

Reply:
This mostly looks good to me. Might be worth adding a `sensitive = true` to the password